travis/maven-settings.xml

```diff
@@ -5,7 +5,7 @@
 
     <servers>
         <server>
-            <id>aws.geodesy.operations</id>
+            <id>awsGeodesyOperations</id>
```

Comment:
Is this id used anywhere? It would be odd if not. Perhaps `mvn deploy` works locally only because mvn picks up your own credentials.

Reply:
No, it seems not used anywhere, so I removed the <server> section. In addition, I add the encripted TRAVIS_TOKEN back to .travis.yml.

Reply:
I have removed maven-settings.xml as it is not used any more.